app/src/main/kotlin/xyz/mustafaali/devqstiles/MainActivity.kt

```diff
@@ -51,4 +98,15 @@ class MainActivity : AppCompatActivity() {
         sendIntent.type = "text/plain"
         startActivity(sendIntent)
     }
+
+    private fun getFeaturesList(): List<Feature> {
+        return listOf(
+                Feature("Toggle USB Debugging", "Enable/disable USB debugging from your notification drawer", R.drawable.ic_toggle_usb_debugging),
+                Feature("Keep Screen On", "Keep screen on when connected via USB, but turn it off when connected to a charger", R.drawable.ic_toggle_keep_screen_on),
+                Feature("Show Touches", "Show touch points when you touch the screen, ideal for demos", R.drawable.ic_toggle_show_taps),
+                Feature("Demo Mode", "Cleans up the status bar for those perfect screenshots", R.drawable.ic_toggle_demo_mode),
+                Feature("Change Animator Duration", "Change the default animator duration to easily debug animations", R.drawable.ic_animator_duration),
+                Feature("Toggle Animation Scale", "Toggle window animation scale, transition animation scale, and animator duration scale", R.drawable.ic_animation)
```

Comment:
Update to `("Toggle all animations", "Enable/disable all animations with one click, perfect for running Espresso tests, ....)`